File: clases 25-11-2025/Tipos de clases c#/clase normal con interfaz/Estudiante.cs
public class Estudiante : IEstudiante
{
    public string Nombre { get; set; }
    public int Grado { get; set; }

    public Estudiante(string nombre, int grado)
    {
        Nombre = nombre;
        Grado = grado;
    }

    public void MostrarInfo()
    {
        Console.WriteLine($"Nombre: {Nombre}, Grado: {Grado}");
    }
}

File: clases 25-11-2025/Tipos de clases GO/clase normal con interfaz/Estudiante.java
public class Estudiante implements IEstudiante {
    public String nombre;
    public int grado;

    // Constructor con parámetros
    public Estudiante(String nombre, int grado) {
        this.nombre = nombre;
        this.grado = grado;
    }

    @Override
    public void mostrarInfo() {
        System.out.println("Nombre: " + nombre + ", Grado: " + grado);
    }
}
